cmd/appengine/device.go

```diff
@@ -43,18 +43,24 @@ const (
 	dataSnapshotCurl = `curl -X GET -H "Accept: application/json" -H "Content-Type: application/json" \
 	-H "User-Agent: astarte-go" \
 	-H "Authorization: Bearer $TOKEN" \
-	"https://$ASTARTE_BASE_URL/appengine/v1/$REALM/devices/$DEVICE_ID/interfaces/$INTERFACE/$PATH?limit=1`
+	"https://$ASTARTE_BASE_URL/appengine/v1/$REALM/devices/$DEVICE_ID/interfaces/$INTERFACE/$INTERFACE_PATH?limit=1`
 
 	getSamplesCurl = `curl -X GET -H "Accept: application/json" -H "Content-Type: application/json" \
 	-H "User-Agent: astarte-go" \
 	-H "Authorization: Bearer $TOKEN" \
-	"https://$ASTARTE_BASE_URL/appengine/v1/$REALM/devices/$DEVICE_ID/interfaces/$INTERFACE/$PATH`
+	"https://$ASTARTE_BASE_URL/appengine/v1/$REALM/devices/$DEVICE_ID/interfaces/$INTERFACE/$INTERFACE_PATH`
 
-	sendDataCurl = `curl -X POST -H "Accept: application/json" -H "Content-Type: application/json" \
+	sendDataCurl = `curl -X PUT -H "Accept: application/json" -H "Content-Type: application/json" \
 	-H "User-Agent: astarte-go" \
 	-H "Authorization: Bearer $TOKEN" \
-	"https://$ASTARTE_BASE_URL/appengine/v1/$REALM/devices/$DEVICE_ID/interfaces/$INTERFACE/$PATH
-	--data '{"data" : $DATA}'`
+	"https://$ASTARTE_BASE_URL/appengine/v1/$REALM/devices/$DEVICE_ID/interfaces/$INTERFACE/$INTERFACE_PATH
+	--data "{\"data\" : $DATA}" `
```

Comment:
This refers now to the `set-property` command, so it would be best to call it `setPropertyCurl`.
The deprecated `send-data` command might refer either to this template or to `sendDatastreamCurl`, both are ok as long as it is specified what template is being used